src/styles/stars.css

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
check 4 point in the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

Reply:
done